Makefile

```diff
@@ -37,8 +41,59 @@ ut-circle: binary
 	--with-xunit --xunit-file=/circle_output/output.xml; RC=$$?;\
 	[[ ! -z "$$COVERALLS_REPO_TOKEN" ]] && coveralls || true; exit $$RC'
 
+dist/calicoctl:
+	mkdir -p dist
+	curl -L http://www.projectcalico.org/latest/calicoctl -o dist/calicoctl
+	chmod +x dist/calicoctl
+
+run-kubernetes-master: run-etcd run-service-proxy binary dist/calicoctl kubectl
+	# Stop any existing kubelet that we started
+	-docker rm -f calico-kubelet-master
+
+	# Remove any pods that the old kubelet may have started.
+	-docker rm -f $$(docker ps | grep k8s_ | awk '{print $$1}')
+
+	# Run the kubelet which will launch the master components in a pod.
+	docker run \
```

Comment:
If we're running the kubelet on the host docker, then the pods that it creates are going to start up using the host docker as well.  This won't work for the multi-node case.

I think we want the kubelet / master services all running on docker in docker, right?